packages/framework/client-logger/app-insights-logger/src/fluidAppInsightsLogger.ts

```diff
@@ -9,6 +9,68 @@ import {
 	type ITelemetryBaseLogger,
 } from "@fluidframework/core-interfaces";
 
+/**
+ * An Enum of Fluid telemetry event categories
+ *
+ * @public
+ */
+export const TelemetryEventCategory = {
+	Performance: "performance",
+	Generic: "generic",
+	Error: "error",
+} as const;
+
+/**
+ * {@inheritDoc (TelemetryEventCategory:variable)}
+ *
+ * @public
+ */
+export type TelemetryEventCategory =
+	(typeof TelemetryEventCategory)[keyof typeof TelemetryEventCategory];
+
+/**
+ * The configuration object for the {@link FluidAppInsightsLogger}
+ *
+ * @public
+ */
+export interface FluidAppInsightsLoggerConfig {
+	/**
+	 * This Configuration defines how filtering will be applied to Fluid telemetry events flowing throguh the logger.
+	 * This determines which events will be sent to Azure App insights.
+	 */
+	filterConfig: {
+		/**
+		 * Determines whether all telemetry events are sent or not sent by default and whether filters will exclude matching telemetry events or include them.
+		 *
+		 * "inclusive" mode means all logs are NOT SENT by default and only the events that match specified filters will be sent (included).
```

Comment:
Let's be clearer here about events being allowed/excluded if they match _any_ of the specified filters or _all_ of the specified filters.